server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -3084,7 +3037,7 @@ static std::vector<sc_synth_definition_ptr> wrapSynthdefs( std::vector<sc_synthd
         wrappedSynthdefs.emplace_back( std::move( ptr ) );
     }
 
-    return std::move( wrappedSynthdefs );
+    return wrappedSynthdefs ;
```

Comment:
don't leave a space before the last semicolon -- if you can `git amend` your commit to fix that, then git history will be preserved when i reformat the codebase